dapp/src/components/buySell/SwapHomepage.js

```diff
@@ -39,12 +39,17 @@ const SwapHomepage = ({
 
   // mint / redeem
   const [swapMode, setSwapMode] = useState(
-    localStorage.getItem(lastSelectedSwapModeKey) || 'mint'
+    typeof window !== 'undefined'
```

Comment:
using `process.browser` is a bit more reliable then checking for presence of `window`